From: Jan Kara <jack@suse.cz>
Subject: block: Fix up kabi after blkcg merge fix
Patch-mainline: Never, kABI fix
References: bsc#1198020

Fix up changed kABI due to additional definitions coming from
linux/blk-cgroup.h.

Signed-off-by: Jan Kara <jack@suse.cz>

---
 block/blk-merge.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/block/blk-merge.c
+++ b/block/blk-merge.c
@@ -7,7 +7,9 @@
 #include <linux/bio.h>
 #include <linux/blkdev.h>
 #include <linux/scatterlist.h>
+#ifndef __GENKSYMS__
 #include <linux/blk-cgroup.h>
+#endif
 
 #include <trace/events/block.h>
 
